integration-tests/validator/src/main/java/org/apache/camel/quarkus/component/validator/it/ValidatorResource.java

```diff
@@ -32,11 +32,28 @@ public class ValidatorResource {
     @Inject
     ProducerTemplate producerTemplate;
 
-    @Path("/xml")
+    @Path("/classpath")
     @POST
     @Consumes(MediaType.APPLICATION_XML)
     @Produces(MediaType.TEXT_PLAIN)
-    public String processOrder(String statement) {
-        return producerTemplate.requestBody("direct:start", statement, String.class);
+    public String processOrderInXmlFromClassPath(String statement) {
+        return producerTemplate.requestBody("direct:classpath", statement, String.class);
     }
+
+    @Path("/filesystem")
+    @POST
+    @Consumes(MediaType.APPLICATION_XML)
+    @Produces(MediaType.TEXT_PLAIN)
+    public String processOrderInXmlFromFileSystem(String statement) {
+        return producerTemplate.requestBody("direct:filesystem", statement, String.class);
+    }
+
+    @Path("/http")
+    @POST
+    @Consumes(MediaType.APPLICATION_XML)
+    @Produces(MediaType.TEXT_PLAIN)
+    public String processOrderInXmlFromHTTP(String statement) {
+        return producerTemplate.requestBody("direct:http", statement, String.class);
+    }
```

Comment:
This test is working in the native mode?

Reply:
@zhfeng yes I tested with `mvn clean install -Pnative` as well and not seeing any failures on native mode. Thanks!

Reply:
Sorry my mistake you are right the http one is failing I can see that now, thanks for pointing out.

Reply:
Added `quarkus.ssl.native=true` for https endpoint, now tests are working in native mode as well, thanks @zhfeng